frontend/pulse3d/src/pages/uploads.js

```diff
@@ -175,12 +301,15 @@ export default function Uploads() {
   useEffect(() => {
     getAllJobs();
     // start 10 second interval
-    const uploadsInterval = setInterval(() => {
+
+    setTimeout(() => {
       // don't call get jobs if downloading ro deleting in progress because it backs up server
-      if (!["downloading", "deleting"].includes(modalState)) getAllJobs();
+      if (!["downloading", "deleting"].includes(modalState)) {
+        toggleUpdateData(!updateData);
+      } //getAllJobs();
     }, [1e4]);
     //clear interval when switching pages
-    return () => clearInterval(uploadsInterval);
+    //return () => clearInterval(uploadsInterval);
```

Comment:
there are two of these set timeouts... I'd keep this one and remove the other one